### PR TITLE
Add stampel to billing expenses pdf

### PR DESCRIPTION
**Pull request by chrysalag**
# [Story in Trello](https://trello.com/c/BLW3R6Vb/13-1-spesenformular-stempel-generierung)

### Done for approval?

* [x] Acceptance criteria are met
* [x] Code quality checks are green
* ~[ ] Readme updated if needed~
* ~[ ] Story under test~
* [x] Mobile works
* ~[ ] Seeds created~
* [x] Translated
* [x] Code is reviewed

### Done after the merge?

* [ ] Deployed to staging after the merge
* [ ] Tested manually on staging

### CODE REVIEW

**Comment by chrysalag** [comment]
Should be merged after #213

**Comment by chrysalag** [comment]
Should be merged after #213

File: app/views/billing_expenses/show.html.slim [hunk 1]
```diff
@@ -71,5 +71,55 @@
...
+                br= @billing_expense.created_at.year
+          tr
+            td
+              p
+                = t('billing_expense_pdf.account')
+                br= '4621'
+            td
+              p
+                = t('billing_expense_pdf.kst')
+                br= '4182'
```
Comment (toupeira) [inline app/views/billing_expenses/show.html.slim]: This actually puts the string inside the `<br>` tag, which is invalid HTML. Split it into two lines:

```slim
br
'4182
```

File: config/locales/de.yml [hunk 2]
```diff
@@ -305,9 +305,19 @@ de:
   billing_expense_pdf:
     <<: *id-generic_keys
     account: Konto
-    cost_center: Kostenstelle
+    amount: Betrag
+    aoz: AOZ
+    book_text_1: Buchungstext
+    book_text_2: Spensenpauschale
```
Comment (toupeira) [inline config/locales/de.yml]: Probably "Spesenpauschale"?

File: config/locales/en.yml [hunk 3]
```diff
@@ -159,17 +159,27 @@ en:
   back: Back
   billing_expense_pdf:
     account: Konto
+    amount: Betrag
+    aoz: AOZ
+    book_text_1: Buchungstext
+    book_text_2: Spensenpauschale
+    chosen_amount: zu überweisender Betrag
```
Comment (toupeira) [inline config/locales/en.yml]: Should be English

Comment (chrysalag) [reply config/locales/en.yml]: This concerns a document they will print, and they use no English documents.

File: app/views/billing_expenses/show.html.slim [hunk 4]
```diff
@@ -79,3 +79,61 @@
             = t_attr(:created_by)
           .col-xs-6
             = @billing_expense.user.profile.contact.full_name
+
+.row
+  br
+  br
+  br
+  br
+  br
```
Comment (casaper) [inline app/views/billing_expenses/show.html.slim]: Thats what CSS is for...

File: app/views/billing_expenses/show.html.slim [hunk 5]
```diff
@@ -79,3 +79,61 @@
...
+  .col-xs-7.col-xs-offset-1
+    .table-responsive
```
Comment (casaper) [inline app/views/billing_expenses/show.html.slim]: What is the need of a responsive table in something you intend to put on a pdf?

File: app/views/billing_expenses/show.html.slim [hunk 6]
```diff
@@ -79,3 +79,61 @@
...
           .col-xs-6
             = @billing_expense.user.profile.contact.full_name
+
+.row
+  br
+  br
+  br
+  br
+  br
+  .col-xs-7.col-xs-offset-1
```
Comment (casaper) [inline app/views/billing_expenses/show.html.slim]: Why not use CSS to have a margin. Bootstrap grid system is good for grid...

File: app/views/billing_expenses/show.html.slim [hunk 7]
```diff
@@ -79,3 +79,61 @@
             = t_attr(:created_by)
           .col-xs-6
             = @billing_expense.user.profile.contact.full_name
+
+.row
+  .col-xs-7.col-xs-offset-1
```
Comment (casaper) [inline app/views/billing_expenses/show.html.slim]: Why use bootstrap grid system in order to get a margin on something that is intended to be on pdf?
And `.table-responsive` has also questionable sense here.

File: app/views/billing_expenses/show.html.slim (hunk 5 above)
Comment (chrysalag) [reply app/views/billing_expenses/show.html.slim]: With show this renders at the browser. The pdf is produced when we press download.

File: app/views/billing_expenses/show.html.slim (hunk 7 above)
Comment (casaper) [inline app/views/billing_expenses/show.html.slim]: Why not use CSS to have a margin. Bootstrap grid system is good for grid...

File: app/views/billing_expenses/show.html.slim (hunk 4 above)
Comment (chrysalag) [reply app/views/billing_expenses/show.html.slim]: That code doesn't exist anymore.

**Pull request merged**